common/meta/api/src/meta_api_test_suite.rs

```diff
@@ -427,10 +428,57 @@ impl MetaApiTestSuite {
                 let err = res.unwrap_err();
                 let err = ErrorCode::from(err);
 
-                assert_eq!(ErrorCode::UnknownDatabase("").code(), err.code());
+                assert_eq!(unknown_database_code, err.code());
+            };
+            // test get table
+            {
+                let got = mt.get_table((tenant, db_name, tbl_name).into()).await;
+                tracing::debug!("get table on unknown db got: {:?}", got);
+
+                assert!(got.is_err());
+                let err = got.unwrap_err();
+                let err = ErrorCode::from(err);
+
+                assert_eq!(unknown_database_code, err.code());
             }
         }
 
+        tracing::info!("--- upsert table on unknown db");
+        {
+            // casually create a upsert table plan
+            // should be not vunerable?
+            let ident = TableIdent::new(114, 514);
+            let plan = UpsertTableOptionReq::new(&ident, "key1", "key2");
+
+            let got = mt.upsert_table_option(plan).await;
+            tracing::debug!("upsert table on unknow db got: {:?}", got);
+
+            assert!(got.is_err());
+            let err = got.unwrap_err();
+            let err = ErrorCode::from(err);
+
+            assert_eq!(unknown_database_code, err.code());
+        }
```

Comment:
I brainlessly wrote similar tests for every operation (`create`, `get`, `upsert` and `drop`). However, the `upsert` operation is seen to operate on tables by `table_id`, regardless of DBs.
So removing this test should solve the problem.